### DISCUSSION TITLE
LXQT spin: Google Chrome won't start after installation

### DISCUSSION THREAD

**Discussion by shaulreznik** [SpiralLinux/SpiralLinux-project#447] [Q&A]
After installing the .deb file and clicking on the icon, the browser doesn't launch:

`[3783:3783:0626/091101.360368:ERROR:browser_main_loop.cc(278)] GLib-GObject: g_type_add_interface_static: assertion 'G_TYPE_IS_INSTANTIATABLE (instance_type)' failed [3783:3783:0626/091101.360403:ERROR:browser_main_loop.cc(278)] GLib-GObject: g_type_interface_add_prerequisite: assertion 'G_TYPE_IS_INTERFACE (interface_type)' failed [3783:3783:0626/091101.360413:ERROR:browser_main_loop.cc(278)] GLib: g_once_init_leave: assertion 'result != 0' failed [3783:3783:0626/091101.360419:ERROR:browser_main_loop.cc(278)] GLib-GObject: g_type_add_interface_static: assertion 'G_TYPE_IS_INSTANTIATABLE (instance_type)' failed [3783:3783:0626/091101.360426:ERROR:browser_main_loop.cc(278)] GLib-GObject: g_type_register_static: assertion 'parent_type > 0' failed`

**Comment by AlphaElwedritsch** [comment]
on mate it is working.
newest updates from today. debian 12.6

Using this repo:

![image](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/ae6a2283-b8ef-4660-b716-828e83174d2d)
